template/eslintrc.js

```diff
@@ -0,0 +1,29 @@
+const config = require('@sumup/foundry/eslint')(
+  {
+    language: 'JavaScript',
+    environments: ['Browser'],
+    frameworks: ['React', 'Emotion', 'Jest'],
+    openSource: false,
+  },
+  {
+    rules: {
+      'react/jsx-uses-react': 'off',
+      'react/react-in-jsx-scope': 'off',
+    },
+    overrides: [
+      {
+        files: [
+          '**/*.story.*',
+          '**/*.stories.*',
+          '**/setupTests.*',
+          '**/test-utils/*',
+        ],
+        rules: {
+          'import/no-extraneous-dependencies': 'off',
+        },
+      },
+    ],
```

Comment:
Do you think we should add these to the default config in Foundry? I feel like our default starter projects should be able to use the default config of our toolkit.